### PR TITLE
feat(content): Expand the edges of the galaxy sprite

### PR DESCRIPTION
**Pull request by Azure3141**
**Content**

## Summary
This pull request completely reworks the Milky Way galaxy sprite, without changing the (relative) positions of any systems or labels. ~~It also adds sprites for two satellite galaxies, the Small and Large Magellanic Clouds.~~

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified

-----------------------

## Feature Details
The new Milky Way sprite incorporates the original, retaining all of the old structural features and bright stars. It is however now a proper circular galaxy, which both looks more aesthetically pleasing and adds additional room to the North and South for expansion (and a proper Map Rework in the future).

Additionally, the Ember Wastes have been given a dull red nebula overlay on the galaxy sprite, making it visually apparent. This provides a fairly large area for future EW content, which can always be expanded if needed.

~~Along with the Milky Way, this PR also introduces our galaxy's two satellite galaxies, the Small and Large Magellanic Clouds. These are essentially directly lifted from the NASA database, with some tweaks to contrast and saturation to fit better with the Milky Way sprite. I've put these in semi accurate locations (okay, the Milky Way's spin is backward so technically they should be on the Eastern side, but that side of the map is already empty enough and it looks better this way), but haven't added any systems or other content to them. They will be hidden due to exploration fog, with no way to reveal them, so I don't think there will be any issues with people thinking they can be reached in the current version of the game.~~

~~Besides replacing the Milky Way sprite, all systems and labels were offset by (112, 22) pixels in the x and y axes to put Sagittarius A* at (0, 0).~~

~~Additionally, while offsetting the coordinates via a script helpfully provided by 1010todd, I decided to round off each coordinate to the nearest whole number. It kind of annoyed me to see system coordinates with three decimals of precision, which is completely unnoticeable on the map. This has literally zero visual impact but it's included here for some unfathomable reason.~~

Satellite galaxies were removed for now, and will be re added if anyone finds a use for them.

## UI Screenshots

Overall view of the Milky Way
image

## Testing Done
I opened the game and was awed to see that the galaxy sprite had indeed loaded, exceeding my wildest expectations (this took me three attempts) [this number has since increased]

## Performance Impact
The new Milky Way sprite is approximately 12 MB, compared to the 1.12 MB of the original galaxy sprite. ~~The Small and Large Magellanic Clouds are about 400 and 700 kB respectively.~~ While this is a significant increase in size, it still has a negligible performance impact.

Thank you to everyone who has participated and given their feedback in the Map Rework Thread! Your feedback and input has been invaluable, and posting my progress on the (many) iterations of this sprite has done a lot to keep me working on it. I would also like to thank ravenshining for inspiring my initial interest in replacing the map sprite a year ago.

### CODE REVIEW

**Comment by Pilcrow182** [comment]
I like this a lot! Very similar to the original, but different enough to just make the galaxy feel much bigger and more alive. Kind of reminds me of those Hubble vs James Webb telescope comparisons, lol.

**Comment by MasterOfGrey** [comment]
I will just straight up say, the sharp edges on the map sprites feel bad.

**Comment by ravenshining** [comment]
So how much of this did you gank from my work in Swizzle 6?

**Comment by quyykk** [comment]
> They will be hidden due to exploration fog, with no way to reveal them, so I don't think there will be any issues with people thinking they can be reached in the current version of the game.

There's a preference to disable the fog of war, so I'm afraid this issue does exist.

**Comment by quyykk** [comment]
I think it would be better to remove, since they'd be unused. When there's a use for them you can PR them

**Comment by Hecter94** [comment]
> Okay I think I fixed the sharp edges by slapping on more vignettes. Also the LMC was overlapping with the Milky Way for some reason, even though I thought I had fixed that earlier, so the most obvious edge is gone now.

Did you update the PR description after making this change? The one shown still has some noticeable sharp edges that look pretty bad.
image

**Comment by roadrunner56** [comment]
I like the new map, but one concern I have is Wanderer space; when going there, the background is very sparse and the hyperlanes are very long, indicating that this is a very spread-out area of the galaxy. With the new map, it looks like it has the same density as Republic space. Perhaps changing wanderer space so that it reflects its new position would be in order.

**Comment by Saugia** [comment]
Similar to roadrunner's concern, I'm also worried about wanderer space, mainly the Pug system. In vanilla, it's actually somewhat hard to see the Pug system unless you click around (and I believe there's another system like this in the south) because they softly blend into a background cloud on the map. That lead to a little mystery as well as reward for finding these systems. 

Also I have to say, I do like the new sprite, however I'm against any map restructuring changes at this time. That should left for a *way* later date, if at all, especially after ones more factions are in the game.

**Comment by roadrunner56** [comment]
I'm personally in favor of reworking the map sooner rather than later. IMO it's much easier to expand the map now than later, when we presumably have way more factions and species in the game. Having to rearrange and deal with those factions will be difficult, and doing it now will save us more work in the future.

**Comment by Zitchas** [comment]
I definitely like this new map; and I also would like to see the map rework sooner rather than later. I'd rather have new races get added where they will ultimately be; rather than adding them to the current map, and once we have it full deciding to move over to a not-slanted map and have to figure out new and revised linkages and/or neighborhoods.

**Comment by Saugia** [comment]
Fair points.

**Comment by ravenshining** [comment]
> I’m pretty sure we talked about your old galaxy texture, which you said you no longer liked and weren’t interested in remaking

I said nothing of the sort.  And yeah you were there and had access to that work, hence the suspicion.  I find it hard to believe you didn't benefit from all that.

Whatever.  Properly attributed or not I'll be glad to see the old, distorted galaxy image go.

And I guess this lacks a lot of the structural features mine had. I'd certainly be interested in collaborating to bring some of those in here.

**Comment by ravenshining** [comment]
"Don't have older revision" is not the same as don't have or plan anything.  I was not aware you were still working on a galaxy sprite, or certainly not at all one that matches the current scale without making room for Grey's Xapeleaux.  I assumed both our efforts were dead with Derpy's disapproval, hence the lack of recent activity there.  Had he at any point expressed a change in heart over the matter of expanding the galaxy sprite, the project would have resumed.

Regardless, this is not the appropriate place to be running court and trotting out discord screenshots.

Your sprite in this PR is certainly an improvement over the current, as I was trying to segue back into.  Personally, I would much prefer to, as I did with mine, add in structure which did not come from a galaxy image, the realistic red streak from M96, and add that bit of room around the core.  There were also some negotiations with Arachi that went into determining the precise edge of Coalition space which may be of value.

**Comment by AbsolutelyLudicrous** [comment]
Very much in love with the new sprite, IMO the galaxy as it is now looks very lopsided.

Re: The Clouds: I doubt anyone will notice them, even with fog-of-war disabled - they're pretty far out. I mean, has anyone noticed the Pleides before the Pug wormhole yet? Plus, their inclusion would allow modders to start developing content earlier for them.

**Comment by LazerLit** [comment]
I really like this new sprite, I do think that the galaxy should be BIG! In a similar vein, I think the Magellanic Clouds should be left in so that content creators can use them.

I also agree with those who have said that if the map is to be reworked, sooner would be better.

**Comment by MasterOfGrey** [comment]
> I'm personally in favor of reworking the map sooner rather than later. IMO it's much easier to expand the map now than later, when we presumably have way more factions and species in the game. Having to rearrange and deal with those factions will be difficult, and doing it now will save us more work in the future.

I am in favor of expanding the map sooner rather than later - I am _**against**_ rearranging current or planned factions, with the 
 possible exception of the Wanderers.

> I definitely like this new map; and I also would like to see the map rework sooner rather than later. I'd rather have new races get added where they will ultimately be; rather than adding them to the current map, and once we have it full deciding to move over to a not-slanted map and have to figure out new and revised linkages and/or neighborhoods.

Ultimately, current and planned races have often been located based on structural features of the underlying galaxy image. The focus on a map update should be on allowing suitable space for additional planning, with meaningful new features. Not on rearranging existing factions.
Tweaking maybe, rearranging, no.
Fortunately, the current implementation looks compatible with this approach - though I don't understand why we felt the need to offset everything just so Sag A was at 0,0.

I would encourage you and Raven to collaborate on adding additional structural features though. I really like where this is at as of right now, but structural features really make faction placement and narrative foundations _pop_.

**Comment by ravenshining** [comment]
For those who aren't on my discord, this is a screenshot of my own map image, in game without any system moves.

Screenshot_20220904_015026

File: copyright [hunk 1]
```diff
@@ -1707,6 +1721,7 @@ Copyright: Saugia (https://github.com/Saugia)
 License: CC-BY-SA-4.0
 Comment: Transparent materials by scrinarii1337.
 
+
```
Comment (beccabunny) [inline copyright:1724]: I really really like it; however, why is this line here?

**Comment by MasterOfGrey** [comment]
> I got started on a slight rework of human space, with the goal of the overall layout remaining the same while systems are organized into clusters separated by both uninhabited systems and distance. This also involved creating more room in the Core for the Xapleaux.

I think you've overdone it with the uninhabited systems in the South and the Rim. They now feel too busy for the relative density of the galaxy in those areas vs other.

I am particularly fond of the Dirt Belt, but I have questions though about the arrangement of pirate systems. Are we missing some and have new ones? Have they just been rearranged?

Otherwise I don't hate this, though I'd be curious to know how much the extra systems in the Syndicate area align with the ones I added.

**Comment by MasterOfGrey** [comment]
> My goal with the Rim was to make it feel more substantial, versus just a chain of fully inhabited systems. I can see some room for making it a bit less dense though (the only systems I added were a couple sticking off the edges, but perhaps they should be spaced more or toned down).

Responded to your specific queries on Discord - but of general note, this is the *second* time that the Rim has been inhabited, and presumably had some attempts at terraforming planets. It's not unreasonable that it would be an unusually habitable string of systems. Especially if there's potential for miscellaneous evidence of the prior civilisation to exist in the uninhabited systems (not on the map) around that chain that would probably show evidence of mining efforts and such that would be easier for the Drak to just hide rather than meticulously comb through them to erase any evidence.

> Just rearranged. The biggest change was moving the Men branch down , which was needed in order to make room to move the Syndicate inwards to make space for the Xapleaux. I also moved Nunki to be linked with Wei, as nothing is really ever done with it in the plot and I felt it was kind of boring for every pirate system to just hang off the bottom of Human space.

Oh *THAT'S* what happened. Moving the Men branch down would definitely mess with the Pirate campaign group's plans for CCOR, but in isolation I don't hate it. Though, per my reasoning in the PR, I'm not sure the Syndicate actually needs to be moved.

> Your Syndicate core region was actually a big inspiration for what I did here, with the chains of uninhabited systems allowing you to get around without passing through inhabited systems. I was originally going to just merge your changes into this (which I really liked and had no issues with), but I was having trouble with the map file so I basically did my own take on it (a lot of the star names are ones you added, with a few exceptions where there were realism issues or duplicate stars under different names). I would have basically no problems with incorporating more of your structure/plans into Syndicate space, as I'm sure you planned it out more thoroughly than I did.

Yeah, there was a specific philosophy of ensuring that there were defined clusters within syndicate space, and perhaps most importantly: *after* my changes two things remained true:
- no new shorter routes from anywhere in Syndicate space to anywhere in Syndicate space were created. (i.e. Sheratan to Markab might be longer now, but the quickest route still goes through all the same systems as it did before.)
- you could get from Stormhold to each cluster of Syndicate space without passing through an inhabited system (by taking the long way through back-systems)

That took some finagling to make happen in a few places.

**Comment by leklachu** [comment]
This has shifted systems in the Pleiades ('pug empty galaxy') relative to the galaxy image

pleiades

**Review by RisingLeaf — APPROVED** [review]
I like this.

**Comment by RisingLeaf** [comment]
On who is this PR waiting in specific, I only see becca in the Reviewers section and they already finished...

**Comment by RisingLeaf** [comment]
> I would like to get more feedback from people, but otherwise I'd guess Derpy wants to review it personally considering how significant of a change it is.

ok, thanks for the reply

File: copyright [hunk 2]
```diff
@@ -1732,6 +1732,20 @@ Files: images/outfits/skadetear*
 Copyright: Anarchist2
 License: CC-BY-SA-4.0
 
+Files:
+	images/ui/milky way*
+	images/ui/large magellanic cloud*
```
Comment (quyykk) [inline copyright:1737]:
```suggestion
 images/ui/milky way*
 images/ui/large magellanic cloud*
```

File: copyright [hunk 3]
```diff
@@ -1732,6 +1732,20 @@ Files: images/outfits/skadetear*
...
+Copyright: Azure
+License: CC-BY-SA-4.0
+Comment: Derived from works by NASA and Michael Zahniser.
+
+Files:
+	images/ui/small magellanic cloud*
```
Comment (quyykk) [inline copyright:1743]:
```suggestion
 images/ui/small magellanic cloud*
```

File: copyright [hunk 4]
```diff
@@ -1732,6 +1732,20 @@ Files: images/outfits/skadetear*
...
+Copyright: NASA
+License: Public-Domain
+ From NASA, and therefore in the public domain because they were created by
+ government employees while doing work for the government.
```
Comment (quyykk) [inline copyright:1747]:
```suggestion
Comment: From NASA, and therefore in the public domain because they were created by
 government employees while doing work for the government.
```

File: data/map.txt [hunk 5]
```diff
@@ -20363,7 +20371,7 @@ system Porrima
 
 system Postverta
 	hidden
-	pos 297 269
+	pos 185 247
```
Comment (warp-core) [inline data/map.txt:20374]: Moving this an associated systems may cause... issues for some people.

File: copyright [hunk 6]
```diff
@@ -760,6 +760,11 @@ Files:
 Copyright: Azure
 License: CC-BY-SA-4.0
 
+Files: images/ui/milky way*
```
Comment (RisingLeaf) [inline copyright:763]:
```suggestion
Files:
  images/ui/milky way*
```

**Review by Amazinite — APPROVED** [review]

**Comment by warp-core** [comment]
Just want to mention that deleting the old sprite might be inconvenient for some plugin authors. I don't know if anyone cares about that.

**Comment by warp-core** [comment]
Can't see the image you've added right now 'cos mobile, but if it's the old sprite, you should also add it's copyright entry back. ~~Or, just call the new one `galaxy`.~~

File: copyright [hunk 7]
```diff
@@ -760,6 +760,12 @@ Files:
 Copyright: Azure
 License: CC-BY-SA-4.0
 
+Files:
+  images/ui/milky way*
```
Comment (Amazinite) [inline copyright:764]:
```suggestion
  images/ui/milky?way*
```
